.gitignore

```diff
@@ -37,7 +37,8 @@ Pods/
 IntegrationTests/CarthageInstallation/Cartfile.resolved
 scan_derived_data/
 generated_docs/
-**/Package.resolved
```

Comment:
Changed it to this, so examples / tests still don't have it any force resolution during CI.